data/os/RedHat.yaml

```diff
@@ -1,17 +1,15 @@
 ---
 oxidized::ruby_dependencies:
-  - rh-ruby23
-  - rh-ruby23-ruby-devel
+  - ruby
 oxidized::install_dependencies:
+  - redhat-rpm-config
```

Comment:
To keep things simpler, I think removing `ruby` from this list makes sense and adding back `libssh2-devel` if it's provided adequately by EPEL, then you can remove `install_dependencies` override from `7.yaml` to avoid duplication.